First pass of the CDC Vaccination Indicator (#1238)
cmu-delphi/covidcast-indicators: cdc_vaccines/README.md

# CDC Vaccinations

> This indicator provides the number official vaccinations in the US. We export the county-level 

Reviewer: ```suggestion
This indicator provides the official vaccination counts in the US. We export the county-level 
```

> daily vaccination rates data as-is, and publishes the result as a COVIDcast signal. 

Reviewer: ```suggestion
daily vaccination rates data as-is, and publish the result as a COVIDcast signal. 
```

> We also aggregate the data to the HHS, State, and Nation levels. 

Reviewer: ```suggestion
We also aggregate the data to the MSA, HRR, State, HHS Region, and Nation levels. 
```

For detailed information see the files DETAILS.md contained in this directory.

Note that individuals could be vaccinated outside of the US. Additionally, 
there is no county level data for counties in Texas and Hawaii. There are unknown 

> counties in each state and a row for unknown county and unknown state.

Reviewer: "There are unknown counties in each state and a row for unknown county and unknown state." 

Can you elaborate on this?

Author: Sorry it wasn't clear! Here is an image from the raw data that better shows what is happening: 
<img width="656" alt="Screen Shot 2021-09-10 at 3 06 59 PM" src="https://user-images.githubusercontent.com/11541884/132904688-4b5944c7-c779-4dd7-9635-0a7270732d3f.png">
These are the unknown counties in each state, and the last row here is an unknown county in an unknown state. They have non-zero values for vaccination.

Reviewer: ```suggestion
there is no county level data for counties in Texas and Hawaii. Each state has some vaccination counts assigned to "unknown county". Some vaccination counts are assigned to "unknown state, unknown county".
```
